client/my-sites/media-library/content.jsx

```diff
@@ -63,12 +64,14 @@ export class MediaLibraryContent extends React.Component {
 		onMediaScaleChange: PropTypes.func,
 		postId: PropTypes.number,
 		isConnected: PropTypes.bool,
+		isMobile: PropTypes.bool,
 	};
 
 	static defaultProps = {
 		mediaValidationErrors: Object.freeze( {} ),
 		onAddMedia: noop,
 		source: '',
+		isMobile: isMobile(),
```

Comment:
This default is not very useful in my opinion. It will be evaluated only once, when the app is loaded, and the result will be the default forever. And won't update when the window is resized.

Reply:
The value of `isMobile` should be retrieved by wrapping the component with the `withMobileBreakpoint` HOC. It will observe window size changes and will automatically pass an always-up-to-date `isBreakpointActive` prop to the wrapped component.